tools/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# From resin/rpi-raspbian:wheezy
+# COPY qemu-arm-static /usr/bin/qemu-arm-static
+
+FROM sedden/rpi-raspbian-qemu:wheezy
```

Comment:
Do you have a plan to use your own raspbian image?
As I think, running ```install-deps-raspberry-pi2_3.sh``` on the general raspbian image(```sedden/rpi-raspbian-qemu```) might be time-consuming.
I recommend that you use **prepared raspbian docker image**, where dependent projects are built and installed.